app/controllers/user_controller.rb

```diff
@@ -1,2 +1,13 @@
 class UserController < ApplicationController
+  def show
+    require 'date'
+    @user = User.find(params[:id])
+  end
```

Comment:
- [ ] You should have an index action in your controller for displaying all the users, [here](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone3/userctr.png) is the requirement